app/controllers/account/charts_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Account::ChartsController < ApplicationController
+  before_action :authorize_admin!
+  def users_by_week
+    chart_data = current_tenant.users
+    render json: chart_data.group_by_week(:created_at, range: time_range).count
+  end
+
+  def companies_by_week
```

Comment:
such suffixes are not relly good, as you may change chart to show monthly or yearly